presto-main/src/main/java/com/facebook/presto/util/PrestoIterators.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.util;
+
+import com.google.common.collect.AbstractIterator;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Iterator;
+
+public class PrestoIterators
```

Comment:
no... I am afraid it might conflict with some other `MoreIterators` in the same way we can conflict when we would name this class `Iterators`. In such case would we need to name `EvenMoreIterators`? IMO `Presto` is nice unique prefix for this project.